rpm/libiphb.spec

```diff
@@ -84,8 +61,6 @@ rm -rf %{buildroot}
 %defattr(-,root,root,-)
 %{_libdir}/libiphb.so.*
 %doc COPYING AUTHORS INSTALL README NEWS ChangeLog
```

Comment:
Something like this perhaps?
%doc COPYING %{?qa_stage_devel:AUTHORS INSTALL README NEWS ChangeLog}